src/PhpWord/Element/Image.php

```diff
@@ -454,7 +454,7 @@ private function setSourceType()
             } else {
                 $this->sourceType = self::SOURCE_GD;
             }
-        } elseif (@file_exists($this->source)) {
+        } elseif ((strpos($this->source, chr(0)) === false) && @file_exists($this->source)) {
```

Comment:
We could use PHP8-functions here, with a polyfill from Symfony.

Reply:
I don't see any new feature in php8 or symfony polyfills to solve this problem.

Reply:
[str_contains](https://wiki.php.net/rfc/str_contains)

https://github.com/symfony/polyfill/blob/master/src/Php80/Php80.php#L91

Reply:
str_contains doesn't solve much... Symfony polyfills require PHP 7, not compatible with PHP 5.x https://github.com/symfony/polyfill/blob/master/src/Php80/composer.json#L23

Reply:
Is that a serious problem? We should drop support for PHP 5 ASAP: https://github.com/PHPOffice/PHPWord/issues/1948

Reply:
Yes, this is problem, because this PR is about compatibility with PHP 8 and PHP 7.4, not about drop support old PHP versions.

Reply:
That's true. I opened an issue, I'd like to hear your opinion :)